src/shogun/regression/LinearRidgeRegression.cpp

```diff
@@ -72,28 +68,16 @@ bool CLinearRidgeRegression::train_machine(CFeatures* data)
     tau_vector.zero();
     tau_vector.add(m_tau);
 
-    Map<MatrixXd> eigen_kernel_matrix(kernel_matrix.matrix, num_feat,num_feat);
-    Map<MatrixXd> eigen_feats_matrix(feats_matrix.matrix, num_feat,num_vec);
-    Map<VectorXd> eigen_y(y.vector, num_feat);
-    Map<VectorXd> eigen_labels(((CRegressionLabels*)m_labels)->get_labels(),num_vec);
-    Map<VectorXd> eigen_tau(tau_vector.vector, num_feat);
-
-    eigen_kernel_matrix = eigen_feats_matrix*eigen_feats_matrix.transpose();
-
-    eigen_kernel_matrix.diagonal() += eigen_tau;
+	linalg::matrix_prod(feats_matrix, feats_matrix, kernel_matrix, false, true);
+	linalg::add_diag(kernel_matrix, tau_vector);
 
-    eigen_y = eigen_feats_matrix*eigen_labels ;
+	auto labels = ((CRegressionLabels*)m_labels)->get_labels();
+	linalg::matrix_prod(feats_matrix, labels, y);
 
-    LLT<MatrixXd> llt;
-    llt.compute(eigen_kernel_matrix);
-    if(llt.info() != Eigen::Success)
-    {
-    	SG_WARNING("Features covariance matrix was not positive definite\n");
-    	return false;
-    }
-    eigen_y = llt.solve(eigen_y);
+	auto decomposition = linalg::cholesky_factor(kernel_matrix);
```

Comment:
why did you change the indentation? Is this the style cheker?

Reply:
Just noticed that the original code is indented with spaces.